spec/encrypt_attributes_spec.rb

```diff
@@ -3,13 +3,15 @@
 class Target
   include EncryptAttributes
   attr_accessor :encrypted_foo,
+                :encrypted_blank,
                 :encrypted_serialized,
                 :encrypted_encoded,
                 :encrypted_serialized_encoded,
                 :encrypted_dynamic_key,
                 :encrypted_create_accessors
 
   encrypted_attribute :foo, secret_key: 'secretkey'
+  encrypted_attribute :blank, secret_key: 'secretkey', allow_empty: false
```

Comment:
The only thing I worry about is that if someone were to remove this attribute they would need a migration to change empty string to nil values.

Reply:
Another/simpler solution would be always treat empty strings as nil in this gem. At the end of the day the whole problem is that AES cannot operate on `blank` values. If you give a malformed value to encrypt, then the gem should just return nothing.